### PR TITLE
Entry: add archived_at property and updateArchived method

### PR DESCRIPTION
**Pull request by sviande**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| CHANGELOG.md  | no
| Fixed tickets | #2716 #3556 
| License       | MIT

Add archived_at date this changes will allow:
* order entries by date of reading
* consistency => tagging or updating some metadata on an entry does'nt change date of reading (third app can rely on this date for stats ex : list all entries read for each week)

### CODE REVIEW

File: src/Wallabag/CoreBundle/Entity/Entry.php [hunk 1]
```diff
@@ -86,6 +86,16 @@ class Entry
...
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="archived_at", type="datetime", nullable=true)
+     *
+     * @Groups({"entries_for_user", "export_all"})
+     */
+    private $archivedAt = null;
+
+
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Entry.php]: This line should be removed

**Comment by j0k3r** [comment]
As it adds new stuff, we need to target the 2.4 branch (which wasn't up to date with the master).
Could you rebase your branch against the upstream/2.4? Thanks.

**Comment by j0k3r** [comment]
I would like to rebase your branch against the updated 2.4 branch.
Could you tick the checkbox on the right sidebar for "Allow edits from maintainers."? Thanks.

**Comment by sviande** [comment]
@j0k3r done

**Comment by j0k3r** [comment]
This is finally merged.
Thanks for your work @sviande 👏

**Pull request merged**